### PR TITLE
Feature/intent tab

### PR DESCRIPTION
**Pull request by oksanasalohubova**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "zenpub",
   "version": "0.1.0",
   "private": true,
-  "homepage": "http://os.reflowproject.eu/",
+  "homepage": ".",
```
Comment (viktorsmari) [inline package.json]: Why this change? :thinking:

File: src/ui/modules/Previews/Intent.tsx [hunk 2]
```diff
@@ -40,6 +40,18 @@ export const Intent: React.FC<Props> = ({
   hideActions,
   onOpen
 }) => {
+  console.log({
```
Comment (viktorsmari) [inline src/ui/modules/Previews/Intent.tsx:43]: Do we need this?

**Review by viktorsmari — CHANGES_REQUESTED** [review]

File: tsconfig.json [hunk 3]
```diff
@@ -28,6 +28,7 @@
     "noEmit": true,
     "strict": true
   },
-  "exclude": ["node_modules", "build", "scripts", "config"],
+  "exclude": ["node_modules",
```
Comment (viktorsmari) [inline tsconfig.json:31]: Maybe skip this change?

File: src/apollo/uploadLink.js [hunk 4]
```diff
@@ -21,7 +21,7 @@ const { extractFiles } = require('extract-files');
  * @kind typedef
  * @name ReactNativeFileSubstitute
  * @type {object}
- * @see [`extract-files` docs](https://github.com/jaydenseric/extract-files#type-reactnativefilesubstitute).
+ * @see [`extract-files` build](https://github.com/jaydenseric/extract-files#type-reactnativefilesubstitute).
```
Comment (viktorsmari) [inline src/apollo/uploadLink.js:24]: Skip this?

File: package.json (hunk 1 above)
Comment (oksanasalohubova) [reply package.json]: returned

**Review by viktorsmari — APPROVED** [review]
